src/factorizations/svd.jl

```diff
@@ -1,207 +1,239 @@
-using LinearAlgebra:
-  LinearAlgebra, Factorization, Algorithm, default_svd_alg, Adjoint, Transpose
-using BlockArrays: AbstractBlockMatrix, BlockedArray, BlockedMatrix, BlockedVector
-using BlockArrays: BlockLayout
-
-# Singular Value Decomposition:
-# need new type to deal with U and V having possible different types
-# this is basically a carbon copy of the LinearAlgebra implementation.
-# additionally, by default we implement a fallback to the LinearAlgebra implementation
-# in hope to support as many foreign types as possible that chose to extend those methods.
-
-# TODO: add this to MatrixFactorizations
-# TODO: decide where this goes
-# TODO: decide whether or not to restrict types to be blocked.
+using MatrixAlgebraKit
```

Comment:
Can you change this to explicit imports, i.e. `using MatrixAlgebraKit: MatrixAlgebraKit, ...`? I imagine we aren't using many names from `MatrixAlgebraKit` anyway.